app/src/main/java/fr/free/nrw/commons/mwapi/OkHttpJsonApiClient.java

```diff
@@ -264,107 +265,54 @@ public OkHttpJsonApiClient(OkHttpClient okHttpClient,
         });
     }
 
-    public Observable<List<Place>> getNearbyPlaces(LatLng cur, String language, double radius)
+    @Nullable
+    public List<Place> getNearbyPlaces(final LatLng cur, final String language, final double radius,
+        final boolean shouldQueryForMonuments)
         throws Exception {
 
         Timber.d("Fetching nearby items at radius %s", radius);
-        String wikidataQuery = FileUtils.readFromResource("/queries/nearby_query.rq");
-        String query = wikidataQuery
+        final String wikidataQuery;
```

Comment:
Choose to go with two different files, so that when WLM is not enabled at all, we don't even include the additional properties needed for WML in the query, so as to not fetch those additional properties when not required at all.

Reply:
Good idea!